source/hardware/controllers/hercules_djcontrol_inpulse_500.rst

```diff
@@ -0,0 +1,316 @@
+   both the gain on the controller’s sound card and in Mixxx.
+
+Mapping description
+-------------------
+
+Decks
+~~~~~
+
+===============================================================  ==========================================
+Control                                                          Function
+===============================================================  ==========================================
+:hwlabel:`SYNC` button                                           Toggle :ref:`Sync Lock <sync-lock>`.
+:hwlabel:`SHIFT` + :hwlabel:`SYNC` button                        Match key to currently playing track.
+:hwlabel:`CUE` button                                            Specifies, plays or recalls temporary cue point.
+:hwlabel:`SHIFT` + :hwlabel:`CUE` button                         Return to the beginning of the track.
+:hwlabel:`Play` button                                           Play/Pause the current track.
+:hwlabel:`SHIFT` + :hwlabel:`Play` button                        Cue Stutter.
+:hwlabel:`VINYL` button                                          Toggle scratch mode (default: on)
+:hwlabel:`SHIFT` + :hwlabel:`VINYL` button                       Toggle vinyl pause effect (default: off)
+Loop :hwlabel:`IN` button                                        Mark start of the loop.
+:hwlabel:`SHIFT` + Loop :hwlabel:`IN` button                     Jump to loop start.
+Loop :hwlabel:`OUT` button                                       Mark end of the loop.
+:hwlabel:`SHIFT` + Loop :hwlabel:`OUT` button                    Jump to loop end.
+Beat Align LED                                                   Track alignment markers (make sure :hwlabel:`BEATMATCH GUIDE` is on for this to work).
+Loop encoder push                                                Reloop toggle
+Loop encoder turn                                                Double or halve current loop
+:hwlabel:`SHIFT` + Loop encoder                                  4-bar loop enable
+:hwlabel:`SLIP`                                                  Toggles splip mode.
+:hwlabel:`QUANT`                                                 Toggles quantize.
+:hwlabel:`SHIFT` + :hwlabel:`QUANT`                              Toggles keylock.
+===============================================================  ==========================================
+
+Browser
+~~~~~~~
+
+===============================================================  ==========================================
+Control                                                          Function
+===============================================================  ==========================================
+:hwlabel:`BROWSER` encoder (turn)                                Move up/down list.
+:hwlabel:`SHIFT` + :hwlabel:`BROWSER` encoder (turn)             Move left/right (fold/unfold) list.
+:hwlabel:`BROWSER` encoder (press)                               Switch focus between list and file view.
```

Comment:
Is this mapped to `MoveFocus|~BackWard|~Forward`?
If yes, it can also focus the Search box and we should mention that, and maybe also may refine the "BROWSER turn" description.